recipe/meta.yaml

```
@@ -264,7 +276,7 @@ outputs:
     version: {{ version }}
     script: build-pyomniscidb.sh
     build:
-      skip: true  # [cuda_compiler_version in ("11.0",)]
+      skip: true  # [cuda_compiler_version in ("11.0",) or not linux64]
```

In general, omniscidb-feedstock provides five packages:
- omniscidb-common
- omniscidb
- pyomniscidb
- omniscidbe
- pyomniscidbe

This PR adds omniscidb with Windows support and ignores other packages (omniscidb-common is incorporated into omniscidb). While omniscidbe and pyomniscidbe can be added in a followup, here we should provide pyomniscidb for Windows as well because windows clients (pyomnisci, for instance) will require it when talking to the omniscidb provided by this PR.

---

Actually, 
```suggestion
      skip: true  # [cuda_compiler_version in ("11.0",)]
```
should add the Windows support to pyomniscidb.

---

done!